pagerdutyplugin/data_source_pagerduty_vendor_test.go

```diff
@@ -27,15 +27,15 @@ func TestAccDataSourcePagerDutyVendor_Basic(t *testing.T) {
 func TestAccDataSourcePagerDutyVendor_ExactMatch(t *testing.T) {
 	dataSourceName := "data.pagerduty_vendor.foo"
 	resource.Test(t, resource.TestCase{
-		PreCheck:     func() { testAccPreCheck(t) },
-		Providers:    testAccProviders,
-		CheckDestroy: testAccCheckPagerDutyScheduleDestroy,
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV5ProviderFactories: testAccProtoV5ProviderFactories(),
+		CheckDestroy:             testAccCheckPagerDutyScheduleDestroy,
 		Steps: []resource.TestStep{
 			{
 				Config: testAccDataSourcePagerDutyExactMatchConfig,
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(dataSourceName, "id", "PKAPG94"),
-					resource.TestCheckResourceAttr(dataSourceName, "name", "Sentry"),
+					resource.TestCheckResourceAttr(dataSourceName, "id", "PAM4FGS"),
+					resource.TestCheckResourceAttr(dataSourceName, "name", "Datadog"),
```

Comment:
Why was this test changed for what is supposed to be a non-breaking refactoring?